whirlpool/oven.py

```diff
@@ -240,7 +239,7 @@ async def set_light(self, on: bool, cavity: Cavity = Cavity.Upper):
                 CAVITY_PREFIX_MAP[cavity]
                 + "_"
                 + ATTR_POSTFIX_LIGHT_STATUS: self.bool_to_attr_value(on)
-            }
+            },
```

Comment:
```suggestion
            }
```

Reply:
Fixed in 046dd236a6a5aec3a2a8bad9062df3d92bc41a0b